jaya-tanwani.md

```diff
@@ -0,0 +1,18 @@
+### JAYA TANWANI
+- Location: Mumbai, Maharashtra, India
+- Bio: A hands-on computer science professional with experience in Machine Learning and Backend development,
+       looking to craft Intelligent and Scalable systems at the fast paced organization.
+- Github: https://github.com/jayatanwani/
+***
+### Blockchain -
+Key developer tools required are:
+1. Geth- It is a gateway to ethereum network also known as a node, using geth we can connect to blockchain, create our own blockchain, create and manage accounts,
+fund with mining or transfer, deploy compiled contracts and, interact with deployed contracts.
+2. Mist - It is a web browser that includes i wallet manager to let you easily create and manage accounts, can connect to mainnet or testnet, create and manage accounts, fund with mining or transfer, compile and deploy contracts, and, interact with deployed contracts.
```

Comment:
It is around this time that you will be asked to choose a password. Couple of things that you must keep in mind here:

Firstly, once set, you will never ever be able to change your password ever again. So you have to set a really strong password from the get go AND you must never forget your password.
Secondly, you will need to enter your password as a final confirmation every single time you want to send Ether.